src/openfermion/measurements/_probability_distributions.py

```diff
@@ -21,17 +21,31 @@
 
 class FourierProbabilityDist(object):
     """
-    Stores a multivariant Fourier representation of a periodic function:
+    Stores an approximate probability distribution of our knowledge of
+    a quantum state in the eigenbasis of a unitary operator.
 
-    f(phi_0,phi_1,...) = sum_j A_j f_j
-    f_j = sum_n (c_{j,2n} cos(n*phi_j) + c_{j,2n-1} sin(n*phi_j))
+    A generic quantum state may be written in the eigenbasis of a unitary
+    operator U,
 
-    In particular, this class stores values of A_j, c_{j,2n},
-    c_{j,2n+1} from the above equation.
+    $|\psi\rangle = sum_j a_j |phi_j\rangle$,
 
-    It also provides routines for multiplying f by functions of the form
-    cos^2(n*phi_j + beta), and for calculating maximum-likelihood
-    distributions of the A_j variables.
+    where psi is the quantum state, phi_j are the eigenstates of U,
+    and a_j are the amplitudes.
+
+    This class stores a distribution P(phi_0, phi_1,... , A_0, A_1,...)
+    (where A_j = |a_j|^2) of our knowledge of these variables which may
```

Comment:
This is confusing. Here, you use `phi_j` as variables, but in the equation above, the `phi_j` label eigenstates, which are not variables.

Reply:
I'm labeling eigenstates by their eigenvalues - U|\phi_j>=e^{i\phi_j}|\phi_j>. Let me make explicit mention of this in the docstring.

Reply:
I see. I would make this clearer by mentioning that the eigenvalues are unknown.

Reply:
Fair enough, I've inserted it.